galaxy/tools/cwl/schema.py

```diff
@@ -68,7 +68,9 @@ def tool(self, **kwds):
             process_definition = self.process_definition(raw_process_reference)
 
         args = {"strict": self._strict}
+        make_tool = kwds.get("make_tool", default_make_tool)
         if LoadingContext is not None:
+            args["construct_tool_object"] = make_tool
```

Comment:
So I thought I tested this - but I guess I must have been taking the other path still :(.